webfonts.list

```diff
@@ -1 +1 @@
-﻿Lato:300,300i,400,400i,600,600i,700,700i,800,800i&subset=cyrillic,cyrillic-ext,greek,greek-ext,latin-ext
+﻿Inter:300,300i,400,400i,600,600i,700,700i,800,800i&subset=cyrillic,cyrillic-ext,greek,greek-ext,latin-ext
```

Comment:
Does this mean we have to replace the Lato font everywhere in our apps by the Inter one? (wondering about Flagship app that embed the Lato file in its bundle)